app/frontend/actions.js

```diff
@@ -645,21 +646,23 @@ export default ({setState, getState}) => {
 
   const sendMaxFunds = async (state) => {
     setState({calculatingFee: true})
-    const maxAmounts = await wallet
+    await wallet
       .getMaxSendableAmount(
         state.sendAddress.fieldValue,
         state.donationAmount.fieldValue !== '',
         state.donationAmount.coins,
         state.checkedDonationType
       )
+      .then((maxAmounts) => {
+        validateAndSetMaxFunds(state, maxAmounts)
+      })
```

Comment:
Same situation as mentioned before, adding a fix to this PR since its related to the network and server error.